test/WebJobs.Script.Tests/Helpers/SimpleWebTokenTests.cs

```diff
@@ -89,6 +90,24 @@ public void Validate_Token_Uses_Website_Encryption_Key_If_Container_Encryption_K
             Assert.True(SimpleWebTokenHelper.TryValidateToken(token, new SystemClock()));
         }
 
+        [Fact]
+        public void Validate_Token_Checks_Signature_If_Signature_Is_Available()
+        {
+            var websiteAuthEncryptionKey = TestHelpers.GenerateKeyBytes();
+            var websiteAuthEncryptionStringKey = TestHelpers.GenerateKeyHexString(websiteAuthEncryptionKey);
+
+            var timeStamp = DateTime.UtcNow.AddHours(1);
+
+            Environment.SetEnvironmentVariable(EnvironmentSettingNames.ContainerEncryptionKey, string.Empty);
+            Environment.SetEnvironmentVariable(EnvironmentSettingNames.WebSiteAuthEncryptionKey, websiteAuthEncryptionStringKey);
+
+            //var token = SimpleWebTokenHelper.CreateToken(timeStamp, websiteAuthEncryptionKey);
```

Comment:
Remove commented out code. I agree with not adding a new CreateToken overload taking the new param.

Reply:
removed!